### PR TITLE
Reduce number of build configs and fix build errors

### PR DESCRIPTION
**Pull request by jeffreyssmith2nd**

### CODE REVIEW

**Comment by tbfleming** [comment]
Is `ripemd.es5.js` an accidental add?

**Comment by jeffreyssmith2nd** [comment]
No, it was something I wanted to add to discuss. With the tsconfig I have here, we don't need babel except for ripemd. If its not something we expect to change, then we can commit the compiled version like I did in this PR and get rid of babel.

If we don't want to go that route, then I'll add babel back to the build process and remove that file.

**Comment by tbfleming** [comment]
I'm ok with this. Move it to `src/` even though it's not really source so it's not at top level.

**Comment by brandonfancher** [comment]
That's awesome, @jeffreyssmith2nd. I agree that it absolutely makes sense to commit the compiled version of this one file in order to avoid the complexity of babel transpilation. 💥

File: src/eosjs2-jsonrpc.ts [hunk 1]
```diff
@@ -135,27 +135,27 @@ export class JsonRpc implements AuthorityProvider {
...
   /** Raw call to `/v1/chain/get_account` */
   public async get_account(accountName: string): Promise<any> {
-    return await this.fetch("/v1/chain/get_account", { accountName });
+    return await this.fetch("/v1/chain/get_account", { account_name: accountName });
   }
 
   /** Raw call to `/v1/chain/get_block_header_state` */
   public async get_block_header_state(blockNumOrId: number | string): Promise<any> {
-    return await this.fetch("/v1/chain/get_block_header_state", { blockNumOrId });
+    return await this.fetch("/v1/chain/get_block_header_state", { block_num_or_id: blockNumOrId });
```
Comment (tbfleming) [inline src/eosjs2-jsonrpc.ts]: Yikes! Someone snuck through a parameter name change in another pull?

File: package.json [hunk 2]
```diff
@@ -8,8 +8,9 @@
     "test": "jest",
     "lint": "tslint -c tslint.json src/**/*.ts",
     "lint-fix": "tslint -c tslint.json src/**/*.ts --fix",
-    "build": "tsc -p ./tsconfig.node.json && cp src/ripemd.js dist && babel dist --out-dir dist",
```
Comment (brandonfancher) [inline package.json]: @jeffreyssmith2nd Now that we're not using babel, can we remove all of the babel-related dependencies now?

File: src/eosjs2-jsonrpc.ts (hunk 1 above)
Comment (tbfleming) [reply src/eosjs2-jsonrpc.ts]: This should get another pull request to fix. The fix is to restore the original names; they shouldn't have changed.

**Comment by jeffreyssmith2nd** [comment]
Dependent on #58 for the build/tests to succeed

**Pull request merged**